pkgs/by-name/db/dbt-duckdb/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+  python3Packages.buildPythonApplication rec {
+    pname = "dbt-duckdb";
+    version = "1.9.6";
+    pyproject = true;
+
+  dependencies = with python3Packages; [
+    pbr
+  ];
+
+  src = fetchFromGitHub {
+    owner = "duckdb";
+    repo = "dbt-duckdb";
+    tag = version;
+    hash = "sha256-zuSOM5Vfr89aVGDcVQtK1Y4c3hRcL8pRscnvpeP6jbU=";
+  };
+
+  build-system = [
+    python3Packages.setuptools
+    python3Packages.wheel
+  ];
+
```

and for the [deps](https://github.com/duckdb/dbt-duckdb/blob/31ddf55161b40960dbc5e85e5a1747a72712a1e7/setup.cfg#L31)
```suggestion
  dependencies = with python3Packages; [
    dbt-common
    dbt-adapters
    duckdb
    dbt-core
  ];
```